scripts/build.d/pybind11

```diff
@@ -10,20 +10,20 @@ if [ "${SRCSYNC}" == "tarball" ] ; then
   pkgver=${VERSION:-2.10.2}
   pkgfull=${pkgname}-${pkgver}
   pkgfn=${pkgfull}.tar.gz
-  pkgurl=https://github.com/pybind/pybind11/archive/refs/tags/v${pkgfull}
+  pkgurl=https://github.com/pybind/pybind11/archive/refs/tags/v${pkgver}.tar.gz
```

Comment:
I think it's a typo for `${pkgfn}`?

Reply:
No. The valid url is
https://github.com/pybind/pybind11/archive/refs/tags/v2.10.2.tar.gz
rather than
https://github.com/pybind/pybind11/archive/refs/tags/pybind11-2.10.2

Reply:
`${pkgfn}` includes the `.tar.gz` extension.

Reply:
Oh yeah. However, the url would still be invalid.

Reply:
Ah, I see.  You are right.  The URL should not use the package name.